test/functional/p2p_unrequested_blocks.py

```diff
@@ -284,11 +284,11 @@ def run_test(self):
             # Only wait a short while so the test doesn't take forever if we do get
             # disconnected
             test_node.sync_with_ping(timeout=1)
-        except AssertionError:
+        except TimeoutError:
```

Comment:
If it is an AssertionError, does it matter if it is no longer processed?

Reply:
When the test is run with `AssertionError` the test exit code seems to be different and the status is
```
 2023-01-20T11:36:31.303000Z TestFramework (WARNING): Timeout. None
2023-01-20T11:36:31.360000Z TestFramework (INFO): Stopping nodes
2023-01-20T11:36:32.191000Z TestFramework (INFO): Cleaning up /var/folders/0y/rhp0p3vx2rj70m8zxhgpw1c80000gn/T/test3ss97r3u on exit
2023-01-20T11:36:32.192000Z TestFramework (INFO): Test timeout
``` 

with `TimeoutError` it is different:
```
2023-01-20T11:38:29.427000Z TestFramework (INFO): Successfully synced nodes 1 and 0
2023-01-20T11:38:29.483000Z TestFramework (INFO): Stopping nodes
2023-01-20T11:38:30.307000Z TestFramework (INFO): Cleaning up /var/folders/0y/rhp0p3vx2rj70m8zxhgpw1c80000gn/T/testvaocqx1o on exit
2023-01-20T11:38:30.307000Z TestFramework (INFO): Tests successful
```